hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/checksum/BaseFileChecksumHelper.java

```diff
@@ -186,6 +207,35 @@ private FileChecksum makeMd5CrcResult() {
         crcPerBlock, fileMD5);
   }
 
+  FileChecksum makeCompositeCrcResult() throws IOException {
+    long blockSizeHint = 0;
+    if (keyLocationInfos.size() > 0) {
+      blockSizeHint = keyLocationInfos.get(0).getLength();
+    }
+    CrcComposer crcComposer =
+        CrcComposer.newCrcComposer(DataChecksum.Type.CRC32, blockSizeHint);
+    byte[] blockChecksumBytes = blockChecksumBuf.getData();
+
+    long sumBlockLengths = 0;
+    for (int i = 0; i < keyLocationInfos.size(); ++i) {
+      OmKeyLocationInfo block = keyLocationInfos.get(i);
+      // For every LocatedBlock, we expect getBlockSize()
+      // to accurately reflect the number of file bytes digested in the block
+      // checksum.
+      sumBlockLengths += block.getLength();
```

Comment:
@jojochuang  `sumBlockLengths` is unused, in hdfs its being used for consumedLastBlockLength calculation.

Reply:
Thanks for checking this out. As far as I understand, Ozone is simpler in how blocks are written, so we shouldn't need to worry about the possibility of mismatch between on-disk length and requested length.

I'll go ahead and remove it.